### PR TITLE
assignee test

### PR DESCRIPTION
**Pull request by noranbear**
assignees

### CODE REVIEW

**Comment by najune** [comment]
1234

**Review by qwaszx357 — APPROVED** [review]
good

**Review by hongjihu — CHANGES_REQUESTED** [review]
request change test

File: README.md [hunk 1]
```diff
@@ -6,3 +6,4 @@
 branch ex
 화이팅
 1233
```
Comment (najune) [inline README.md:8]: 좋아요

**Review by najune — COMMENTED** [review]
1234

**Pull request merged**